packages/app-store/jitsivideo/lib/VideoApiAdapter.ts

```diff
@@ -31,4 +43,16 @@ const JitsiVideoApiAdapter = (): VideoApiAdapter => {
   };
 };
 
+function generateEventSlug(pattern: string, event: CalendarEvent): string {
+  pattern
```

Comment:
We are not using the return value here. So, this fn doesn't do it's job.

Also, I would like to discuss more about what variables need to be supported here and why we even need this?

The way I understand the Jitsi URL is automatically generated. Why would anyone care if it follows a certain format?

@chief-nerd Could you remove the slug pattern implementation from this PR for now?

Reply:
@hariombalhara make the changes.

> The way I understand the Jitsi URL is automatically generated. Why would anyone care if it follows a certain format?

Cause it looks nice :) - especially on my locally hosted version
https://meet.mydomain.com/e8d513ef-b385-48de-8c06-6a483e78e224
vs
https://meet.mydomain.com/Meet-and-Greet-Max-Musterman